app/services/update_user.rb

```diff
@@ -5,14 +5,17 @@ def initialize(user:, attributes:)
   end
 
   def call
-    result = user.update!(
+    user.assign_attributes(
       attributes.except(
         :phone_id, :phone, :phone_confirmed_at,
         :otp_make_default_number
       ),
     )
+
+    user.encrypt_and_digest_password! if attributes.key?(:password)
+    user.save!
     manage_phone_configuration
-    result
+    user
   end
```

Comment:
I think that it might be time for us to consider removing `UpdateUser` entirely, but that's a story for another time :]

Reply:
Yeah, I agree